### DISCUSSION TITLE
Planning For 2025

### DISCUSSION THREAD

**Discussion by yutingzhao1991** [ant-design/ant-design-web3#1306] [General]
What do you expect Ant Design Web3 to support in 2025? Welcome to give us your feedback~
你期望 Ant Design Web3 在 2025 年支持什么？欢迎给我们你的反馈~

**Comment by gin-lsl** [comment]
相似地址高亮功能。一个很实用的小功能，如果 antd web3 来做的话，开发者用起来会更方便点。
目前我在本地已经简单实现了下，如果确定做的话，我再提pr：
![image](https://github.com/user-attachments/assets/bd2cfbf8-c788-4633-9abe-c98ccad60a6a)

**Reply by yutingzhao1991** [reply]
可以考虑。这个是什么场景需要用？设计师有定义使用的场景吗，主要考虑是否具备通用性

**Reply by gin-lsl** [reply]
多用于显示交易列表的场景，区块浏览器基本都会需要类似功能。不是设计师的需求。

**Reply by yutingzhao1991** [reply]
但是需要设计师设计一下，确保通用并符合设计规范，这样才能复用得起来
